src/Razor/Microsoft.AspNetCore.Razor.Language/src/Legacy/CSharpCodeParser.cs

```diff
@@ -1417,6 +1418,22 @@ private void ParseExtensibleDirective(in SyntaxListBuilder<RazorSyntaxNode> buil
                                     return;
                                 }
                                 break;
+
+                            case DirectiveTokenKind.Boolean:
```

Comment:
What does the tooling experience currently look like?
Need to update [DesignTimeDirectiveTargetExtension](https://github.com/dotnet/aspnetcore/blob/master/src/Razor/Microsoft.AspNetCore.Razor.Language/src/Extensions/DesignTimeDirectiveTargetExtension.cs#L63) to provide the appropriate design time support so that it suggests `true` and `false` as options.

Reply:
Ok, good point!